doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1550 @@
+      * The :c:func:`lte_lc_periodic_search_request`, :c:func:`lte_lc_periodic_search_clear`, :c:func:`lte_lc_periodic_search_get`, and :c:func:`lte_lc_periodic_search_set` functions require the new :kconfig:option:`CONFIG_LTE_LC_PERIODIC_SEARCH_MODULE` Kconfig option to be enabled.
+      * The :c:enumerator:`LTE_LC_EVT_PSM_UPDATE` event and the :c:func:`lte_lc_psm_param_set`, :c:func:`lte_lc_psm_param_set_seconds`, :c:func:`lte_lc_psm_req`, :c:func:`lte_lc_psm_get`, and :c:func:`lte_lc_proprietary_psm_req` functions require the new :kconfig:option:`CONFIG_LTE_LC_PSM_MODULE` Kconfig option to be enabled.
+      * The :c:enumerator:`LTE_LC_EVT_MODEM_SLEEP_EXIT_PRE_WARNING`, :c:enumerator:`LTE_LC_EVT_MODEM_SLEEP_ENTER`, and :c:enumerator:`LTE_LC_EVT_MODEM_SLEEP_EXIT` events require the new :kconfig:option:`CONFIG_LTE_LC_MODEM_SLEEP_MODULE` Kconfig option to be enabled.
+      * The :c:enumerator:`LTE_LC_EVT_TAU_PRE_WARNING` event requires the new :kconfig:option:`CONFIG_LTE_LC_TAU_PRE_WARNING_MODULE` Kconfig option to be enabled.
+
+  * Deprecated:
+
+    * The :c:macro:`LTE_LC_ON_CFUN` macro.
+      Use the :c:macro:`NRF_MODEM_LIB_ON_CFUN` macro instead.
+    * The :c:func:`lte_lc_factory_reset` function.
+      Use the ``AT%XFACTORYRESET`` AT command instead.
+      Refer to the :ref:`migration guide <migration_2.8>` for more details.
+    * The :c:enum:`lte_lc_factory_reset_type` type.
+    * The :c:func:`lte_lc_reduced_mobility_get` and :c:func:`lte_lc_reduced_mobility_set` functions.
+      Refer to the :ref:`migration guide <migration_2.8>` for more details.
+    * The :c:enum:`lte_lc_reduced_mobility_mode` type.
+      Refer to the :ref:`migration guide <migration_2.8>` for more details.
+
+  * Removed:
+
+    * The ``lte_lc_init`` function.
+      All instances of this function can be removed without any additional actions.
+    * The ``lte_lc_deinit`` function.
+      Use the :c:func:`lte_lc_power_off` function instead.
+    * The ``lte_lc_init_and_connect`` function.
+      Use the :c:func:`lte_lc_connect` function instead.
+    * The ``lte_lc_init_and_connect_async`` function.
+      Use the :c:func:`lte_lc_connect_async` function instead.
+    * The ``CONFIG_LTE_NETWORK_USE_FALLBACK`` Kconfig option.
+      Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
+      In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
+
+* :ref:`lib_location` library:
+
+  * Fixed:
+
+    * A bug causing the GNSS obstructed visibility detection to sometimes count only part of the tracked satellites.
+    * A bug causing the GNSS obstructed visibility detection to be sometimes performed twice.
```

Comment:
```suggestion
    * An issue causing the GNSS obstructed visibility detection to sometimes count only part of the tracked satellites.
    * An issue causing the GNSS obstructed visibility detection to be sometimes performed twice.
```

Reply:
discussed offline and decided to keep it as is.